### PR TITLE
RDBC-469 We need to send a file cert using pfx12_file 

### PR DESCRIPTION
**Pull request by poissoncorp**
https://issues.hibernatingrhinos.com/issue/RDBC-469

### CODE REVIEW

**Comment by poissoncorp** [comment]
Working on writing tests for that

File: pyravendb/connection/requests_executor.py [hunk 1]
```diff
@@ -39,7 +39,11 @@ def initialize_certificate(certificate):
             return certificate, None
         pfx = certificate["pfx"]
         password = certificate.get("password", None)
-        adapter = Pkcs12Adapter(pkcs12_data=pfx, pkcs12_password=password)
+        adapter = (
+            Pkcs12Adapter(pkcs12_filename=pfx, pkcs12_password=password)
+            if os.path.isfile(pfx)
```
Comment (arekpalinski) [inline pyravendb/connection/requests_executor.py:44]: Shouldn't `if` be one line up?

Comment (poissoncorp) [reply pyravendb/connection/requests_executor.py:44]: Nope, it's Python [tenary](https://book.pythontips.com/en/latest/ternary_operators.html)

**Comment by poissoncorp** [comment]
i'm gonna open this one for v5.0

**Pull request closed without merging**